Database_Structures.md

```diff
@@ -45,3 +45,23 @@ fk_questionid|
 option_content|
 picture_link|
 qustion_multiplier|
+
+USER|
+-|
+id|
+email|
+username|
+registered_at|
+
+TOKEN|
```

Comment:
Token tables are completely unnecessary for authorization

Reply:
All right, then I will remove this schema

Reply:
How are we going to implement 'Remember Me' function then?

Reply:
API should set a bigger expiration time on jwt payload once remember me option is true. This might require a change in contract

Reply:
There could be refresh tokens but that's a huge overhead for us

Reply:
I like the idea of not using the database for this, thanks for the explanation.